index.js

```diff
@@ -13,6 +16,8 @@
 // *** otherwise, the error will stop the rest of your code from running
 // console.log("%cQuestion 2", "color: red") 
 
+const name = "Julio"
```

Comment:
Great! The key here was using `const` because `const` doesn't allow us to reassign our variable